### DISCUSSION TITLE
About Godruoyi | 连波的闲谈杂鱼

### DISCUSSION THREAD

**Discussion by giscus** [godruoyi/gblog#10] [Announcements]
# About Godruoyi | 连波的闲谈杂鱼

Godruoyi，15 年毕业于西南大学计算机专业，Laravel 爱好者；目前靠 Go 养家糊口，抽空学习 Rust，全职远程中

https://godruoyi.com/posts/about-godruoyi/

**Comment by ycy1** [comment]
你好

**Reply by godruoyi** [reply]
hello

**Comment by ycy1** [comment]
hello 在astro 看到你的博客模板 借用一下

**Comment by herijaya** [comment]
test sir

**Comment by keepchen** [comment]
你好啊，欢迎互换友链🍻：  
博客：
https://blog.keepchen.com  
Github:  
https://github.com/keepchen  
Twitter:
https://x.com/wowgogoing

**Reply by keepchen** [reply]
话说，你的白天黑夜模式切换的效果蛮酷的！

**Reply by godruoyi** [reply]
参考的是 https://theme-toggle.rdsx.dev/，对应的 PR 在这里 https://github.com/godruoyi/gblog/pull/19 😊。

**Comment by ycy1** [comment]
哈哈&nbsp; 使用的是 tailwindcss 全局切换的

**Reply by godruoyi** [reply]
hey @ycy1 我感觉你对别人的回复会出现在我的博客评论上，你是不是忘记设置[这里](https://github.com/godruoyi/gblog/blob/astro/src/config.ts#L56-L64)的 ID 了，可能需要你参考这里的注释给自己的博客重新设置 giscus 配置哦。

**Comment by RonHaiT** [comment]
在家里打开你的网站 总感觉有点卡卡的

**Reply by godruoyi** [reply]
不是感觉，就是卡卡的，我太菜了引入的那个丝滑滚动的 lib 没有调教好，等我抽空看看，谢谢你反馈了。

**Comment by bigmanBass666** [comment]
哇哦

**Reply by godruoyi** [reply]
哇哦2

**Comment by echo-LuGuang** [comment]
远程工作对英文语言是不是有要求呢

**Reply by godruoyi** [reply]
有点要求但我觉得要求也不算太高了，因为基本都是异步沟通所以口语差点问题也不大。
